qcengine/programs/empirical_dispersion_resources.py

```diff
@@ -880,6 +897,59 @@ def get_params(entry: dict, base: dict, defaults: list) -> dict:
 
 dashcoeff["d4bjeeqatm"]["definitions"].update(_get_d4bj_definitions())
 
+try:
+
+    def _get_d3_definitions(dashlevel: str) -> dict:
+        """
+        Lucky for us dftd3 provides access to all damping parameters directly.
+
+        However, we are not happy with the structure of the dict and
+        therefore do some restructuring. The below solution tries to bend
+        the data returned by dftd3 into the right shape to make qcng happy.
```

Comment:
thanks. and sorry for the historical necessity.

Reply:
At least I learned a something about the things one can do with dict-comprehension in Python.